package/utils/rbextract/Makefile

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (C) 2010 Gabor Juhos <juhosg@openwrt.org>
+# Copyright (C) 2018 Chris Schimp <silverchris@gmail.com>
+# Copyright (C) 2019 Robert Marko <robimarko@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rbextract
+PKG_RELEASE:=1
+CMAKE_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/rbextract
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=Boot Loaders
+  TITLE:=RouterBOOT ART Extractor
+  DEPENDS:=@TARGET_ipq40xx||TARGET_ath79 +liblzo
```

Comment:
Why TARGET_ipq40xx (I would only use ath79 for now and add ipq40xx when we use it there)?
If you keep it, I assume you mean `@(TARGET_ipq40xx||TARGET_ath79)`

Reply:
Sorry, I missed this one; you had already pointed it.